emails/models.py

```diff
@@ -309,6 +312,14 @@ def emails_replied(self):
             )
         return total_num_replied + self.num_email_replied_in_deleted_address
 
+    @property
+    def level_one_trackers_blocked(self):
+        return (
+            sum(ra.num_level_one_trackers_blocked for ra in self.relay_addresses)
+            + sum(da.num_level_one_trackers_blocked for da in self.domain_addresses)
+            + self.num_level_one_trackers_blocked_in_deleted_address
```

Comment:
Because these can be `null`, we need something like:

```suggestion
            sum(ra.num_level_one_trackers_blocked or 0 for ra in self.relay_addresses)
            + sum(da.num_level_one_trackers_blocked or 0 for da in self.domain_addresses)
            + self.num_level_one_trackers_blocked_in_deleted_address or 0
```